Index: deploy.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#!/usr/bin/env sh\n\n# 确保脚本抛出遇到的错误\n\nset -e\n\n# 生成静态文件\n\nnpm run docs:build\n\n# 进入生成的文件夹\n\ncd docs/.vuepress/dist\n\ngit init\n\ngit add -A\n\ngit commit -m 'deploy'\n\n# 如果发布到 https://<USERNAME>.github.io/<REPO>\n\ngit push -f git@github.com:ikecoolon/ikecoolon.github.io.git main\n\ncd -
===================================================================
diff --git a/deploy.sh b/deploy.sh
--- a/deploy.sh	
+++ b/deploy.sh	
@@ -20,6 +20,6 @@
 
 # 如果发布到 https://<USERNAME>.github.io/<REPO>
 
-git push -f git@github.com:ikecoolon/ikecoolon.github.io.git main
+git push -f git@github.com:ikecoolon/ikecoolon.github.io.git demo
 
 cd -
\ No newline at end of file
Index: deploy-other.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#!/usr/bin/env sh\n\n# 确保脚本抛出遇到的错误\n\nset -e\n\n# 生成静态文件\n\n#npm run docs:build\n\n# 进入生成的文件夹\n\ncd docs/.vuepress/dist\n\ngit init\n\ngit add -A\n\ngit commit -m 'deploy'\n\n# 如果发布到 https://<USERNAME>.github.io/<REPO>\n\ngit push -f git@github.com:ikecoolon/ikecoolon.github.io.git main\n\ncd -
===================================================================
diff --git a/deploy-other.sh b/deploy-other.sh
--- a/deploy-other.sh	
+++ b/deploy-other.sh	
@@ -20,6 +20,6 @@
 
 # 如果发布到 https://<USERNAME>.github.io/<REPO>
 
-git push -f git@github.com:ikecoolon/ikecoolon.github.io.git main
+git push -f git@github.com:ikecoolon/ikecoolon.github.io.git demo
 
 cd -
\ No newline at end of file
